packages/style-engine/src/styles/typography/fontSize.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import type { Style, StyleOptions } from '../../types';
+
+const fontSize = {
+	name: 'fontSize',
+	generate: ( style: Style, options: StyleOptions ) => {
```

Comment:
Since most of the new styles share the same generate logic, it might save a few bytes to create a general `getGenerateCallback` utility method.

Premature optimization? You decide!